pkg/fileutil/fileutil_test.go

```diff
@@ -166,3 +168,45 @@ func TestDirPermission(t *testing.T) {
 		t.Errorf("expected error, got nil")
 	}
 }
+
+func TestRemoveMatchFile(t *testing.T) {
+	tmpdir := t.TempDir()
+	defer os.RemoveAll(tmpdir)
```

Comment:
No need to manually remove the `tmpdir`, because it is automatically removed by the golang testing utilities.
FYI. 
* https://github.com/golang/go/blob/go1.16.15/src/testing/testing.go#L911-L915 
* https://github.com/golang/go/blob/go1.16.15/src/testing/testing.go#L957
* https://github.com/etcd-io/etcd/pull/14248

Reply:
addressed.